### DISCUSSION TITLE
Assioma Uno - do I need to double the power manually?

### DISCUSSION THREAD

**Discussion by holger8080** [doudar/SmartSpin2k#181] [Q&A] [answered]
Could someone please confirm: 
if I use the Assioma Unos (one-sided-only power meter pedals) as power meter, do I need to set PowerCorrectionFactor=2 to get the correct total power?

**Comment by fireyeti** [comment]
The Assioma should report power correctly to the SmartSpin2k. Certain power tap pedals were reporting single side power.

**Comment by drz01** [comment, marked as answer]
The current release has a specific switch to double power for one-sided pedals; this is being expanded in development branches to PCF for other use cases as well. The output can also be doubled directly in firmware via the Favero app. Established software such as Zwift should automatically differentiate Uno from Duo -- but only in a direct connection and not via SS2k.
